### ISSUE TITLE
Identity UI lacks aria-required annotations

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#25867]
Originally reported as an accessibility issue at https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1163969. I verified this does repro.

The identity UI has various forms (e.g., register new user) in which fields are required. However it doesn't have the `aria-required="true"` attribute on those `<input>` elements, so screen readers don't announce them as required.

**Comment by HaoK** [comment]
Keeping this open until scaffolding PR is merged

**Comment by HaoK** [comment]
@deepchoudhery @vijayrkn is the scaffolding repro ready for 6.0 targeted changes yet?

**Comment by deepchoudhery** [comment]
not yet but very soon. I'll ping you here.

**Comment by deepchoudhery** [comment]
@HaoK master branch is ready for 6.0.0 changes.
